### PR TITLE
Improvement of the Presto Example :

### PR DESCRIPTION
**Pull request by JeanLescut**
- Simplified a lot the whole script
- Made everything compatible with Python 3
- Removed the print() for proper logger
- Added assert() for parameters sanity checks
- Allows passing keywords arguments to pd.read_csv()
- Removed useless dependencies
...

### CODE REVIEW

**Comment by JeanLescut** [comment]
If you have any questions on my PR, feel free

File: example/qubole_presto_api_example.py [hunk 1]
```diff
@@ -1,68 +1,79 @@
...
-    if query is None or query == "":
-        return None
-    cmd = PrestoCommand.create(query=query)
-    query_id = str(cmd.id)
-    print "Starting Command with id: " + query_id + "\nProgress: =>",
+#######################
+# PRIVATE FUNCTIONS
+#######################
+def _wait_for_cmd(cmd, update=True):
     while not PrestoCommand.is_done(cmd.status):
```
Comment (msumit) [inline example/qubole_presto_api_example.py]: won't this get stuck for forever here, if `update` is `False`?

Comment (JeanLescut) [reply example/qubole_presto_api_example.py]: yes that's true. I've just changed it to include a timeout

Comment (JeanLescut) [reply example/qubole_presto_api_example.py]: thanks

**Comment by JeanLescut** [comment]
Hey guys,
I have:
- rebased my fork on the current "unreleased" branch of your repo
- commited and pushed a new version, with some minor code improvement and a timeout (to answer msumit question

Now, even if Qubole query got stuck, the function still return with error.
Feel free to improve it as well.

**Comment by JeanLescut** [comment]
Also, it seems that Travis CI is failing to install Python 3.3 and 2.6 (??) but I don't think it's related to this PR

**Comment by chattarajoy** [comment]
yes @JeanLescut  looks like the Ubuntu version was not pinned in our configuration and hence it got updated to 16.XX, will fix it. Thanks for the PR btw

**Comment by chattarajoy** [comment]
@JeanLescut I have fixed the issue, please rebase your branch once more and squash all your commits.

**Pull request closed without merging**